### DISCUSSION TITLE
Diffuse Reflectance in time domain

### DISCUSSION THREAD

**Discussion by zrostamp** [fangq/mcx#197] [Help]
I am trying to calculate the diffuse reflectance on one detector from the semi-infinite medium. There is an example that calculates the diffuse reflectance in the steady state( drefmc = mcxcwdref(detp, cfg))
Now I am confused about how I can use it in the time domain. 
I was wondering if you could clarify this problem for me.

Thanks

**Comment by fangq** [comment]
see my reply in the mailing list: https://groups.google.com/g/mcx-users/c/5gTfeh76_cE
